modules/afsocket/afsocket-source.c

```diff
@@ -691,6 +879,40 @@ afsocket_sd_save_listener(AFSocketSourceDriver *self)
     }
 }
 
+static void
+afsocket_sd_save_dynamic_window_ctr(AFSocketSourceDriver *self)
+{
+  GlobalConfig *cfg = log_pipe_get_config(&self->super.super.super);
+
+  if (self->connections_kept_alive_across_reloads)
+    {
+      cfg_persist_config_add(cfg, afsocket_sd_format_dynamic_window_pool_name(self),
+                             self->dynamic_window_ctr, (GDestroyNotify) dynamic_window_pool_unref, FALSE);
+    }
+  else
+    {
+      dynamic_window_pool_unref(self->dynamic_window_ctr);
+    }
+
+  self->dynamic_window_ctr = NULL;
+}
+
+static gboolean
+afsocket_sd_restore_dynamic_window_ctr(AFSocketSourceDriver *self)
```

Comment:
`ctr` should be renamed to `pool`. (5/7)

Reply:
This remained the same after the fixup commits. Also please fix the `afsocket_sd_save_dynamic_window_ctr()` function name please (I missed that in my last review :( )